src/Components/Greetings/Greetings.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+
+function Greetings(props) {
+    const {lang, children} = props;
+    switch(lang){
+        case "en":
+            lang = "hello";
+            break;
+            case "fr":
+                lang = "bonjour";
+                break;
+                case "es":
+                    lang = "holla";
+                    break;
+default :
+return "oi";
```

Comment:
You should not reassign values to a prop that you are receiving. Also, you are trying to do it on a const variable meaning that it's impossible to reassign